integration/airflow/openlineage/airflow/adapter.py

```diff
@@ -194,7 +194,7 @@ def _build_run(
                 "nominalTime": NominalTimeRunFacet(nominal_start_time, nominal_end_time)
             })
         if parent_run_id:
-            facets.update({"parentRun": ParentRunFacet.create(
```

Comment:
For backward compatibility, we need to publish both facets for some period

Reply:
That makes sense. Let me update it.